screens/DriverSearch.1.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
```

Comment:
What's the point of this file? If you aren't using this page, delete it. You shouldn't have numbers in file names. So if this is an updated version of the DriverSearch, update the original, don't make new files.